Source/scf/scf_relax.cpp

```diff
@@ -620,7 +610,8 @@ Castro::do_hscf_solve()
 
         for (int lev = 0; lev <= finest_level; ++lev) {
 
-            const Real* dx = parent->Geom(lev).CellSize();
+            auto geomdata = parent->Geom(lev).data();
+            const Real* dx = geomdata.CellSize();
```

Comment:
This should probably be CellSizeArray for safety.

Reply:
done